include/spirv-tools/libspirv.h

```diff
@@ -53,6 +53,7 @@ typedef enum spv_result_t {
   SPV_ERROR_INVALID_CFG = -11,
   SPV_ERROR_INVALID_LAYOUT = -12,
   SPV_ERROR_INVALID_CAPABILITY = -13,
+  SPV_ERROR_INVALID_DATA = -14,
```

Comment:
This new case should be documented. (Sorry the previous ones are all lacking explanation; it's kinda legacy and oversight.)